src/main/java/org/tub/vsp/data/container/analysis/StreetAnalysisDataContainer.java

```diff
@@ -0,0 +1,42 @@
+package org.tub.vsp.data.container.analysis;
+
+import org.tub.vsp.computation.Modifications;
+import org.tub.vsp.computation.NkvCalculator;
+import org.tub.vsp.data.container.base.StreetBaseDataContainer;
+
+import java.util.LinkedHashMap;
+import java.util.SequencedMap;
+
+public class StreetAnalysisDataContainer {
+    private final StreetBaseDataContainer streetBaseDataContainer;
+    private final SequencedMap<String, Double> nkvByChange = new LinkedHashMap<>();
+
+    public StreetAnalysisDataContainer(StreetBaseDataContainer streetBaseDataContainer) {
+        this.streetBaseDataContainer = streetBaseDataContainer;
+        this.addComputations(streetBaseDataContainer);
+    }
+
+    private StreetAnalysisDataContainer addNkvByChange(String change, Double nkv) {
+        nkvByChange.put(change, nkv);
+        return this;
+    }
+
+    private void addComputations(StreetBaseDataContainer streetBaseDataContainer) {
```

Comment:
Hier fügen werden die Berechnungen den Objekten hinzugefügt, die wir in der csv datei ausgeben. Das kann beliebig erweitert werden. Ich war mir nicht sicher, 
* ob wir nur die nkv methode verwenden wollen oder auch die andere
* welchen Wert die zusätzlichen Fzkm haben sollen